recipes-core/initrdscripts/files/init-readonly-rootfs-overlay-boot.sh

```diff
@@ -61,6 +61,10 @@ read_args() {
 				ROOT_RWRESET=$optarg ;;
 			rootrwoptions=*)
 				ROOT_RWMOUNTOPTIONS_DEVICE="$optarg" ;;
+			overlaymodule=*)
+				OVERLAY_MODULE="$optarg"
```

Comment:
OVERLAY_MODULE is not used. I think we can remove it. Sorry about my last comment.